h/groups/models.py

```diff
@@ -58,6 +59,24 @@ def slug(self):
     def __repr__(self):
         return '<Group: %s>' % self.slug
 
+    def documents(self, db, limit=25):
```

Comment:
For an instance method, use `sqlalchemy.orm.object_session` to retrieve the session, don't pass it in.